content/doc/book/managing/cli.adoc

```diff
@@ -185,14 +185,20 @@ Authorities:
 %
 ----
 
-
 == Using the CLI client
 
 While the SSH-based CLI is fast and covers most needs, there may be situations where the CLI
 client distributed with Jenkins is a better fit. For example, the default transport for the CLI client
 is HTTP which means no additional ports need to be opened in a firewall for its
 use.
 
+==== Comparing SSH and CLI client
+
+Both SSH and jenkins-cli.jar provide access to a set of commands that lets you interact with Jenkins from a command line, but they have a few differences:
+
+* Jenkins SSH does not require any custom jar file on the client side, making it easier to access Jenkins from a variety of sources
+* Jenkins CLI client can be intelligent, performing file access and other interactive processing, whereas SSH client is dumb. Therefore, some commands can only run through Jenkins CLI and not via SSH
```

Comment:
My concern here is with the verbiage. I am not a fan of terms such as `dumb`. I would prefer this read something like `whereas the SSH client is not as intuitive`
Thoughts?

Reply:
humm. I guess it is a valid suggestion. What about: 

"SSH client was build to be a generic tool to serve several propouses, so it don't offer no easy-way to do basic things that are common and specific to jenkins enviroment. Therefore use Jenkins CLI maybe will increase your productivity and your user experience."

I'm not a english native speaker, so please help me if you find any sintaxe or semantic error. ;)

Reply:
I like that very much. Thank you 🙂

Reply:
Did it. Thank you.